### PR TITLE
LUN-524: Close dropdown when clicking outside

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by LeandraH — COMMENTED** [review]
Works nicely except when I tap on any other interactive element (the field to edit the word, the button to record an audio, the button to save), then the dropdown stays open.

**Comment by SteffiStuffel** [comment]
> Works nicely except when I tap on any other interactive element (the field to edit the word, the button to record an audio, the button to save), then the dropdown stays open.

You are right, thank you. Fixed it.

**Review by sarahsporck — CHANGES_REQUESTED** [review]
Tested in emulated Android and simulated ios. When now trying to select an article nothing is selected.

**Comment by SteffiStuffel** [comment]
I had a closer look at this, and think the best is to replace the Dropdown library with an other dropdown library. Unfortunately styling needs to be done with StyleSheets. Feel free to suggest better options, as I am aware this is not perfect, buy I couldn't find any better solution.

**Review by LeandraH — APPROVED** [review]
Looks good, tested in emulated iOS. I don't have a better idea for the StyleSheets either :(

File: src/components/Dropdown.tsx [hunk 1]
```diff
@@ -1,90 +1,64 @@
...
   placeholder: string
-  items: ItemType<string | number>[]
-  itemKey: string
-  multiSelect?: boolean
-  setValue: Dispatch<SetStateAction<T | null>>
-  value: T | null
-  schema?: { label: string; value: string }
+  items: Array<DropdownItemType>
+  setValue: Dispatch<SetStateAction<number | null>>
+  value: string | null
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/components/Dropdown.tsx:41]: Yes, I need to fix this

File: src/components/Dropdown.tsx [hunk 2]
```diff
@@ -1,90 +1,64 @@
...
+    paddingHorizontal: theme.spacingsPlain.sm,
+  },
+  placeholderStyle: {
+    fontSize: theme.fonts.defaultFontSizeWithoutUnit,
+    color: theme.colors.placeholder,
+  },
+  selectedTextStyle: {
+    fontSize: theme.fonts.defaultFontSizeWithoutUnit,
+  },
+  iconStyle: {
```
Comment (sarahsporck) [inline src/components/Dropdown.tsx:30]: This is required so that the icon is rendered on ios

**Review by sarahsporck — APPROVED** [review]
I installed pods for you. Android works fine :)

**Pull request merged**